mw-webapp/src/logic/wayPage/labels/AdjustLabelsModalContent.tsx

```diff
@@ -106,24 +106,27 @@ export const AdjustLabelsBlock = (props: JobTagsProps) => {
       }
 
       {props.isEditable &&
-        <Modal
-          isOpen={isJobDoneModalOpen}
-          content={
-            <PromptModalContent
-              placeholder={LanguageService.way.filterBlock.jobTagPlaceholder[language]}
-              close={() => setIsJobDoneModalOpen(false)}
-              onOk={createJobTag}
-              okButtonValue={LanguageService.modals.promptModal.okButton[language]}
-              cancelButtonValue={LanguageService.modals.promptModal.cancelButton[language]}
-            />
-          }
-          trigger={
-            <Button
-              value={LanguageService.way.filterBlock.addTagButton[language]}
-              onClick={() => setIsJobDoneModalOpen(true)}
-            />
-          }
-        />}
+        <div className={styles.wrapperButton}>
```

Comment:
1. Bad naming, if name wrapperButton this div should be around button but not around all modal
2. I thimk that better to add this class inside modal component for get rid the same situation in the future

@tritonJS826  What do you think?

Reply:
fixed